function.rb

```diff
@@ -0,0 +1,9 @@
+
+def function(n)
+  if n == 0
+    1
+  else
+    n * function(n-1)
+  end
+end
```

Comment:
Consider if n < 0 or n is not an integer

Reply:
```suggestion
def function(n)
  if n == 0
    1
  else
    n * function(n-1)
  end
end
```

Can you make it more clear what this function is attempting to do?